src/interface/file.rs

```diff
@@ -268,18 +268,18 @@ impl EmulatedFileMap {
         // set the file equal to where were mapping the count and the actual map
         let _newsize = f.set_len((COUNTMAPSIZE + mapsize) as u64).unwrap();
 
-        let map : Vec::<u8>;
-        let countmap : Vec::<u8>;
+        let map : Vec<u8>;
+        let countmap : Vec<u8>;
```

Comment:
I think we should leave this change for now, since we're still sorting out our code style guidelines for Rust.  Certainly a bunch of this will get modified as part of [another issue](https://github.com/Lind-Project/lind_project/issues/373).  Let's leave the minor style changes which we may or may not end up making out of here for now.